runtime/src/bank.rs

```diff
@@ -1761,4 +1770,42 @@ mod tests {
         assert!(!bank5.is_in_subtree_of(2));
         assert!(!bank5.is_in_subtree_of(4));
     }
+
+    #[test]
+    fn test_bank_inherit_tx_count() {
+        let (genesis_block, mint_keypair) = GenesisBlock::new(500);
+        let parent = Arc::new(Bank::new(&genesis_block));
+
+        // Bank 1
+        let bank = Arc::new(new_from_parent(&parent));
+        // Bank 2
+        let bank2 = Arc::new(new_from_parent(&bank));
```

Comment:
Following the naming convention from the other tests, the first item could be named either `bank` or `bank0`, but this second item would always be `bank1`.